src/get_manifest.ts

```diff
@@ -1,15 +1,19 @@
 import { deepMerge, getProtocolInterface, path } from "./deps.ts";
-import { getDefaultExport } from "./utilities.ts";
+import {
+  forEachValidatedManifestFunction,
+  getDefaultExport,
+} from "./utilities.ts";
 
 // Responsible for taking a working directory, and an output directory
 // and placing a manifest.json in the root of the output directory
 
 /**
  * Returns a merged manifest object from expected files used to represent an application manifest:
- * `manifest.json`, `manifest.ts` and `manifest.js`.
+ * `manifest.json`, `manifest.ts` and `manifest.js`. If both a `json` and `ts`/`js` are present, then
+ * these `ts`/`js` export will be merged over the `json` file.
  * @param {string} cwd - Absolute path to the root of an application.
  */
-export const createManifest = async (cwd: string) => {
+export const getManifest = async (cwd: string) => {
```

Comment:
Renamed this method from `create` to `get` as there is no creation or side effect that happens from this method; I think it's a bit clearer otherwise it may leave the impression we are e.g. writing files or something.